R/install-dev.sh

```diff
@@ -38,7 +38,12 @@ pushd $FWDIR > /dev/null
 if [ ! -z "$R_HOME" ]
   then
     R_SCRIPT_PATH="$R_HOME/bin"
-   else
+  else
+    # if system wide R_HOME is not found, then exit
+    if ! [ `command -v R` ]; then
```

Comment:
I just tried on my local machine (I removed R installation), and either way is working, giving me the same message
```
RENs-MacBook-Pro:spark xin$ ./R/install-dev.sh
Cannot find 'R_HOME'. Please specify 'R_HOME' or make sure R is properly installed.
```

if `[ ! ` is a better practise I can change to it, I'm just know very basic bash :P